interfaces/repo.go

```diff
@@ -80,6 +80,112 @@ func (r *Repository) AddInterface(i Interface) error {
 	return nil
 }
 
+// QueryOptions describes options for QueryInterfaces.
```

Comment:
I agree about connections, not just the new API end-point and new snap subcommand but also as a general term.

I think I applied all the suggestions below. Let me know if I missed anything.